.github/workflows/windows.yml

```diff
@@ -47,6 +47,13 @@ jobs:
     - name: Enter DevShell
       run: 'libva-utils\.github\workflows\EnterDevShell.ps1 ${{ inputs.architecture }}'
       shell: pwsh
+    - name: Build DirectX-Headers
+      run: |
+        git clone https://github.com/microsoft/DirectX-Headers -b v1.606.4 --depth 1

```

Comment:
Hardcoding `v1.606.4` is not good. We will need to support that somehow and I've no idea how. Can we just go with tip of the 'main' branch?

Reply:
To simplify this and remove the explicit cloning from the CI, I changed the dependency to use the [meson built-in subprojects feature](https://mesonbuild.com/Subprojects.html).
Instead of cloning it manually in the CI pipeline, meson will try to resolve the `directx-headers` dependency as a package in the system it's building on. If the package is not found, it automatically falls back to cloning/building from the `DirectX-Headers` repo.

![image](https://user-images.githubusercontent.com/82116493/197609156-87c6b8d0-cb4c-40ae-a723-3757d3ad3e94.png)